src/snowflake/cli/plugins/nativeapp/commands.py

```diff
@@ -137,7 +137,7 @@ def app_run(
         is_interactive=is_interactive,
     )
     return MessageResult(
-        f"Your application ({processor.app_name}) is now live:\n"
+        f"Your application object ({processor.app_name}) is now live:\n"
```

Comment:
I'm not 100% sure the context of this message, but the term "live" makes me thing that it's available to the world. In this case, it's really just that the application object is created in the provider's account, no? Maybe say, "now available"? Just a thought.

Reply:
Updating this to "available" in the next commit.